project/MiMaFilters.scala

```diff
@@ -22,4 +22,7 @@ object MiMaFilters {
     ProblemFilters.exclude[MissingClassProblem]("scala.caps$Pure"),
     ProblemFilters.exclude[MissingClassProblem]("scala.caps$unsafe$"),
   )
+  val TastyCore: Seq[ProblemFilter] = Seq(
+    ProblemFilters.exclude[MissingMethodProblem]("dotty.tools.tasty.TastyFormat.APPLYsigpoly"),
```

Comment:
I think this isn't necessary on the main branch now that the base version is 3.3.0, mima shouldn't complain about new APIs.

Reply:
Ah true